### ISSUE TITLE
A little meddling on the sendwhatmsg() in the CountryCodeExceptionErr

### ISSUE THREAD

**Issue by oamuoyiemamamin** [Ankit404butfound/PyWhatKit#22]
This maybe not a big change but I think it may be helpful for a lot of people like me. I am Maram Shrijan and I'm 14 years old. I would like to request a change in the sendwhatmsg function particularly the CountryCodeExeption. 
First of all, the CountryCodeExeption is raised whenever there is no "+" in the string. This is going to be a problem for us who are using excel files to store phone numbers because Excel assumes it as a number.
After some research I found out that this function is using the URL to navigate to the particular number to send the WhatsApp Web message. The URL not only accepts "+" but also '(underscore)' as some formatting.
For example:
This is what we must format our format our phone numbers currently : "+60**********"
Here is what the URL also accepts : "60_***********"
So therefore here's what I propose for the CountryCodeExeption:
`    if "+" not in phone_no and "_" not in phone_no:
raise CountryCodeException("Country code missing from phone_no")
    timehr = time_hour`

original:
`    if "+" not in phone_no :
        raise CountryCodeException("Country code missing from phone_no")
    timehr = time_hour
`

This way, people who use excel to store phone numbers are not going to have that much of a problem.

Yes, I did edit the code I downloaded. I hope the fix could come soon.
BTW That little delay at the back is really useful

**Comment by Ankit404butfound** [comment]
Hello, I will try to remove this country code exception by using some API to get phone numbers' country code, till then what you can do is to save number like this in excel sheet "_60*****" and then when reading the file for sending message do this
```
num = get_num_from_excel()
num = num.replace("_","+")
sendwhatmsg(num......)
```

**Comment by aaryanrr** [comment]
Hey @oamuoyiemamamin, this correction has been implemented and will be released with version 5.2. Thank you for pointing this out.
